payloads/v10/channel.ts

```diff
@@ -1919,17 +1922,30 @@ export interface APIBaseAutoPopulatedSelectMenuComponent<
  *
  * String Selects can be configured for both single-select and multi-select behavior. When a user finishes making their choice(s) your app receives an interaction.
  *
- * String Selects must be placed inside an Action Row and are only available in messages. An Action Row can contain only one select menu and cannot contain buttons if it has a select menu.
+ * An Action Row can contain only one select menu and cannot contain buttons if it has a select menu.
  *
  * @see {@link https://discord.com/developers/docs/components/reference#string-select}
  */
-export interface APIStringSelectComponent extends APIBaseSelectMenuComponent<ComponentType.StringSelect> {
+export type APIStringSelectComponent = APIStringSelectComponentInActionRow | APIStringSelectComponentInModal;
+
+export interface APIStringSelectComponentInActionRow extends APIBaseSelectMenuComponent<ComponentType.StringSelect> {
 	/**
 	 * Specified choices in a select menu; max 25
 	 */
 	options: APISelectMenuOption[];
 }
 
+export interface APIStringSelectComponentInModal extends APIBaseSelectMenuComponent<ComponentType.StringSelect> {
+	/**
+	 * Specified choices in a select menu; max 25
+	 */
+	options: APISelectMenuOption[];
+	/**
+	 * Whether this component is required
+	 */
+	required?: boolean;
+}
+
```

Comment:
Like I said in https://canary.discord.com/channels/222078108977594368/999796393810067607/1402933854477619290, unify in one interface please